bundles/nl.esi.comma.testspecification/src/nl/esi/comma/testspecification/abstspec/generator/AssertionsHelper.java

```diff
@@ -94,6 +95,8 @@ static String expression(Expression expression, Function<String, String> variabl
 			return String.format("'%s'", pexpr.getValue());
 		} else if (expression instanceof ExpressionConstantReal e) {
 			return Double.toString(e.getValue());
+		} else if (expression instanceof ExpressionEnumLiteral e) {
+			return Integer.toString(e.getLiteral().getValue().getValue());
```

Comment:
The `value` of an enum literal is optional, so this query might result in a `NullPointerException` when not set. In the case of not set, the int  value should be the index of the literal in its enumeration. So I would have expected a helper function.